tools/rna_tools/vienna_rna/kinfold.xml

```diff
@@ -0,0 +1,105 @@
+]]>
+    </command>
+
+    <inputs>
+        <param format="*" name="input" type="data" label="Fasta file"/>
+        <param argument="--Temp" type="float" value="37.0" label="temperature [°C]"/>
+        <param argument="--dangle" type="select" label="how to treat dangling end energies">
+            <option value="0">0: none</option>
+            <option value="2" selected="true">2: double</option>
+            <option value="1">1: normal</option>
+        </param>
+        <param argument="--logML" type="boolean" truevalue="--logML" falsevalue="" checked="false" label="Logarithmic energy functions for multi-loops"/>
+		
+		<section name="move_set_options" title="Move set options">
+			<param argument="--noShift" type="boolean" truevalue="--noShift" falsevalue="" checked="false" label="Turn of shift moves."/>
+			<param argument="--noLP" type="boolean" truevalue="--noLP" falsevalue="" checked="false" label="Forbid structures containing isolated base-pairs"/>
+		</section>
+		
+		<section name="simulation_options" title="Simulation options" >
+			<param argument="--num" type="integer" value="1" label="Number of trajectories to compute" help="" optional="true"/>
+			<param argument="--time" type="float" value="500.0" label="Set maximum length of folding trajectory" help="The default (500) is very short and meant for testing purposes only." optional="true"/>
+			<param argument="--grow" type="float" value="0" label="Simulate folding during transcription with a chain growth event taking place every rate timesteps." help="" optional="true"/>
+			<param argument="--glen" type="integer" value="15" label="Start a folding during transcription simulation with an inital chain length of len."  help="" optional="true" />
+			<param argument="--fpt" type="boolean" truevalue="--fpt" falsevalue="" checked="true" label="compute first passage time" help="stop when a stop-structure is reached"/>
+			<param argument="--met" type="boolean" truevalue="--met" falsevalue="" checked="false" label="Use the Metropolis rule for rate between two neighboring conformations" help="Use the Metropolis rule for rate between two neighboring conformations, i.e. k=min{1,exp(−dE/RT)}. By default Kinfold uses the symmetric Kawasaki rule k=exp(−dE/2RT)."/>
+		</section>
+		
+		<section name="output_options" title="Output Options">
+			<param argument="--lmin" type="boolean" truevalue="--lmin" falsevalue="" checked="false" label="Don’t print complete trajectory, but only local minimas encountered" help=""/>
+			<param argument="--cut" type="float" value="20.0" label="Cutoff energy" help="Print only those parts of the trajectory that stays below energy." optional="true"/>
+		</section>
+       
+    </inputs>
+    <outputs>
+        <data format="txt" name="output"/>
+        <data format="log" name="log"/>
+    </outputs>
+    <tests>
+        <test>
+            <param name="input" value="kinfold_input.txt"/>
+            <output name="output">
+				<assert_contents>
```

Comment:
I guess I know what the problem with indentation is ... please configure your editor to use 4 spaces instead of a TAB.